src/index.css

```diff
@@ -228,4 +231,14 @@ footer p {
 
 footer p {
   margin: 0;
-}
\ No newline at end of file
+}
+
+footer .image-wrapper {
```

Comment:
<img width="191" alt="Screenshot 2023-08-02 at 2 54 05 PM" src="https://github.com/FCCColumbus/cbus-web/assets/22714448/150c4fd4-bc50-43fe-aa00-164f221d6129">

Nit: fix formatting, remove extra whitespace.

Reply:
<img width="401" alt="Screenshot 2023-08-02 at 2 56 17 PM" src="https://github.com/FCCColumbus/cbus-web/assets/22714448/914a01d7-ba31-4ba6-9c29-ad2a980af948">

fwiw I only first noticed this by doing a local diff of your branch against the `upstream/dev` branch (after making sure to pull in order to be up-to-date). You can see the red in the console, indicating there's extra whitespace. (Not all terminals will be configured to show this necessarily; it's probably a default setting that comes with macOS Xcode tools or some other terminal settings I have applied.)